pkg/storage/system_intake_test.go

```diff
@@ -637,20 +637,20 @@ func (s *StoreTestSuite) TestFetchSystemIntakeMetrics() {
 			1,
 		},
 	}
-	for _, tt := range completedTests {
```

Comment:
Did the underscored code trip the linter?

Reply:
It was memory aliasing in loop by taking address of `tt`

https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop. It looks like this issue gets resolved in Go 1.22 though

Reply:
Ah, because of the `&` below, got it. So the linter is working as intended here. Nice!